app/dashboard/_components/RiskAndOpps/RiskAndOpportunitiesSection.tsx

```diff
@@ -0,0 +1,16 @@
+import Section from '@/app/dashboard/_components/RiskAndOpps/Section'
+import { risksData } from '@/app/dashboard/_components/RiskAndOpps/data/RisksData'
+import { opportunitiesData } from '@/app/dashboard/_components/RiskAndOpps/data/OpportunitiesData'
+
+const RiskAndOppsSection = () => {
+  return (
+    <div className='col-span-6 md:col-span-6 lg:col-span-4'>
+      <Section title='Risks' items={risksData} />
+      <div className='mt-8'>
```

Comment:
there is no need for an extra div here, you can just put space-y-8 on the div on line 7 and remove this div